### DISCUSSION TITLE
How to use CRACO to configure "React Refresh Babel transform" plugin?

### DISCUSSION THREAD

**Discussion by glomotion** [dilanx/craco#517] [Q&A]
We're trying to configure cypress after recently upgrading craco, and are stuck on this error.

Is there anyone who could help me work out how to use craco to configure the plugin in question to `{skipEnvCheck: true}`?

```bash
Error: [BABEL] /****/demo.test.cy.tsx: React Refresh Babel transform should only be enabled in development environment. Instead, the environment is: "test". If you want to override this check, pass {skipEnvCheck: true} as plugin options. (While processing: "/*****/node_modules/react-refresh/babel.js")
```

**Comment by nf3** [comment]
check out https://github.com/dilanx/craco/issues/522
"build": "cross-env REACT_APP_FAST_REFRESH=false FAST_REFRESH=false craco --max_old_space_size=6000 build"
